src/api/libcellml/componententity.h

```diff
@@ -313,11 +313,12 @@ class LIBCELLML_EXPORT ComponentEntity: public NamedEntity
 
     bool doEquals(const EntityPtr &other) const override; /**< Virtual implementation method for equals, @private. */
 
-    ComponentEntity(); /**< ComponentEntity Constructor. */
+    class ComponentEntityImpl; /**< Forward declaration for pImpl idiom, @private. */
 
-private:
```

Comment:
Why is the `private` section gone?

(The descriptions below reference `@private`...?)

Reply:
2cdf2f8